lib/types/items.d.ts

```diff
@@ -0,0 +1,64 @@
+import { IRazorpayQuery } from "./api";
+
+export interface IRazorpayItem {
+    name: string;
+    description?: string;
+    amount: number;
+    currency: string;
+    active?: boolean;
+  }
+
+  export interface IRazorpayItemId extends IRazorpayItem{
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Lets rename this to IRazorpayItem

Reply:
Done